ml-proto/Makefile

```diff
@@ -21,7 +20,7 @@ OCB =		ocamlbuild $(OCB_FLAGS)
 .PHONY:		all opt unopt
 unopt:		$(NAME_UNOPT)
 opt:		$(NAME_OPT)
-all:		opt unopt test
+all:		winmake.bat opt unopt test
```

Comment:
What constrains this to only invoking winmake on Windows? I don't see anything that does